Dockerfile.e2e

```diff
@@ -2,13 +2,16 @@ FROM cypress/included:9.7.0
 
 WORKDIR /e2e
 
-COPY apps/user-office-frontend-e2e/cypress.json ./user-office-frontend-e2e/
-COPY apps/user-office-frontend-e2e/package* ./user-office-frontend-e2e/
-COPY apps/user-office-frontend-e2e/tsconfig.json ./user-office-frontend-e2e/tsconfig.json
-COPY apps/user-office-frontend-e2e/wait-for-frontend.sh ./user-office-frontend-e2e/
-COPY apps/user-office-frontend-e2e/cypress ./user-office-frontend-e2e/cypress/
+COPY apps/user-office-frontend-e2e/cypress.json .
+COPY apps/user-office-frontend-e2e/package.json .
+COPY apps/user-office-frontend-e2e/package-lock.json .
+COPY apps/user-office-frontend-e2e/tsconfig.json .
+COPY apps/user-office-frontend-e2e/cypress ./cypress/
+COPY --chmod=755 apps/user-office-frontend-e2e/wait-for-frontend.sh .
```

Comment:
Rather than have the dockerfile set the execute permissions at build time, could we set and commit the execute permissions to git https://stackoverflow.com/a/38285462/16235117